packages/kbn-visualization-utils/index.ts

```diff
@@ -11,3 +11,5 @@ export { getTimeZone } from './src/get_timezone';
 export { getLensAttributesFromSuggestion } from './src/get_lens_attributes';
```

Comment:
Not related to this file/line, but to the overall behaviour:
If I run the following query in Discover
```
FROM kibana_sample_data_logs | STATS AVG(bytes), MAX(bytes) BY machine.os.keyword ,tags.keyword | LIMIT 1000
```
I can't change anymore the resulting stacked bar chart into an heatmap in this PR. If I try to change it, no error appear, but the chart doesn't change to heatmap.

It seems to work fine in `main`.

Reply:
could be caused by the fact that you have changed the `enum` and the `Heat map` chart type in lens doesn't correspond anymore to the `Heatmap` ChartType?

Reply:
No it is unrelated, it looks for both the visualization id and the title

Reply:
Yes nice catch, I was very optimistic to remove some code from the histogram but I can't do it. Bringing this code back fixes these problems.

https://github.com/elastic/kibana/pull/197453/commits/ea3e44335d5435208a68ea74f49ef2d643f32ca7

in your case the api won't return the heatmap (it is also marked with the yellow dot so we need this code for the cases where the suggestions api do not return the type)